concent_api/core/views.py

```diff
@@ -653,6 +659,78 @@ def handle_send_force_subtask_results_results_response(request, client_message):
     return HttpResponse("", status = 202)
 
 
+def verify_message_subtask_results_accepted(subtask_results_accepted_list):
```

Comment:
Please add typing notation and short comment what this function does and returns.